conboarding.md

```diff
@@ -39,6 +39,17 @@ How much does it cost to run things?
 
 
 1. Send desired login name and .pub portion of the SSH key to Yarik for development box(es): smaug, typhon, etc
+   1. login to `smaug`: `ssh -i /path/to/key me@smaug.dartmouth.edu -p
+      $SOME_PORT`
+   1. login to `typhon`: `ssh -i /path/to/key me@typhon.dartmouth.edu -p
+      $SOME_PORT`
+      Note: When loging in from campus (use `eduroam`), SSH does not always work on
+      `typhon`. Instead it is recommended to use `ssh-agent` and forward the authentication connection using `-A`
+         1. (Assuming `ssh-agent` is running) `ssh-add /path/to/key`
+         1. `ssh-add -l` should now show your fingerprint.
+         1. SSH into `smaug` with connection forwarding: `ssh -A me@smaug.dartmouth.edu -p $SOME_PORT`
+         1. Once on `smaug` `ssh-add -l` should now show the same fingerprint.
+         1. From `smaug`, proceed to `typhon` with `ssh me@typhon.dartmouth.edu -p $SOME_PORT`
```

Comment:
you might benefit from specifying some details within your ~/.ssh/config for the given host(s) 

1.  such as `SOME_PORT` - so you don't need to enter it, 
2. `AgentForward` to not need `-A`
3. `ProxyJump` to not need to jump manually.  So e.g.

```
Host smaug smaug.dartmouth.edu drogon drogon.dartmouth.edu typhon typhon.dartmouth.edu
  Port $SOME_PORT
  ForwardAgent yes

Host typhon typhon.dartmouth.edu
  ProxyJump smaug.dartmouth.edu
```
and then just `ssh typhon.dartmouth.edu` would jump you over through smaug.